devops/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+﻿vagrant plugin install sahara
+
+$currentDirectory = Split-Path $MyInvocation.MyCommand.Path
+$sourceDir = "chocolatey-test-environment"
+$repoUrl = "https://github.com/chocolatey-community/chocolatey-test-environment.git"
+
+# exit if directory already exists
+if ($(test-path $sourceDir -pathtype container) -eq $true)
+{
+	write-error "Cannot clone repo, folder '$sourceDir' already exists."
+	exit 1
+}
+
+# clone repo
+git clone $repoUrl
+ 
+cd $sourceDir
```

Comment:
Push-Location would be what's used here and you would definitely Pop-Location afterwards - it's an anti-pattern to change directories on a user.